examples/index.md

```diff
@@ -389,127 +342,61 @@ Content-Type: application/vnd.api+json
-This example with a missing `auth_key` parameter was chosen over an example with an invalid
-URI query parameter, since it may make sense to ignore invalid parameters in some cases, and
-have the request succeed rather than fail. As described in the [Bulk
-extension](http://jsonapi.org/extensions/bulk/#bulk-operations):
-
-> ...a request MUST completely succeed or fail (in a single "transaction").
-> ...any request that involves multiple operations MUST only succeed if all operations are performed
-successfully
-
-Examples of invalid parameters may be `?page[prev]=` (parameter present, but has no value),
-`?include=auther` (invalid parameter value, perhaps a typo), `?felds[author]=`,
-(invalid parameter key, perhaps a typo), `?redirect_to=http%3A%2F%2Fwww.owasp.org` (invalid
-parameter, in this case, a phishing attack), etc.
-
-### Using JSON API extensions
-
-Request and response using the [JSON PATCH
-Extension](http://jsonapi.org/extensions/jsonpatch/):
-
-```http
-PATCH /posts/1
-Content-Type: application/vnd.api+json; ext=jsonpatch
-Accept: application/vnd.api+json; ext=jsonpatch
-
-[ { "op": "remove" } ]
-```
-
-```http
-HTTP/1.1 422 Unprocesssable Entity
-Content-Type: application/vnd.api+json; ext=jsonpatch
-
-[
-  {
-    "errors": [
-      {
-        "status": "422",
-        "detail": "Operation missing required keys: 'path'"
-      }
-    ]
-  }
-]
-```
-
-Notice that JSON PATCH is a special semantics and does not mean an HTTP PATCH request
-containing JSON.
+#### Invalid Query Parameters
 
-Request and response using the [Bulk extension](http://jsonapi.org/extensions/bulk/)
+The `source` member can also be used to indicate that the error originated
+from a problem with a URI query parameter, like so:
 
 ```http
-PATCH /photos
-Content-Type: application/vnd.api+json; ext=bulk
-Accept: application/vnd.api+json; ext=bulk
-
-{
-  "data": [{
-    "type": "articles",
-    "id": "1",
-    "attributes": {
-      "title": "To TDD or Not"
-    }
-  }, {
-    "type": "articles",
-    "id": "2",
-    "attributes": {
-      "title": "LOL Engineering"
-    }
-  }]
-}
+GET /api/posts/1?include=auther
 ```
 
 ```http
 HTTP/1.1 400 Bad Request
 Content-Type: application/vnd.api+json
 
-
 {
   "errors": [
     {
-      "status": "404",
-      "source": { "pointer": "/data/articles/1" },
-      "detail": "Article with id 1 could not be found."
+      "source": { "parameter": "include" },
+      "title":  "Invalid Query Parameter",
+      "detail": "The resource does not have an `auther` relationship path."
     }
   ]
 }
 ```
 
-Since requests are transactional, as the [Bulk
-extension](http://jsonapi.org/extensions/bulk/#bulk-operations) describes: *"The
-state of the server MUST NOT be changed by a request if any individual operation fails."*
-Thus, even though one of the objects to update was found, one was not, so none get updated.
+In most cases, JSON API requires the server to return an error when it encounters
```

Comment:
This is a revised version of the note that was here before, just clarifying the difference between API-specific query params (which may often be ignored) and JSON API–defined ones, which usually can not be.